### PR TITLE
Expand the installation instructions

### PR DESCRIPTION
**Pull request by kumar303**
This provides some detailed instructions for how to install each example. It also shows how to use the web-ext tool to install examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,22 @@ To check the minimum version of Firefox needed for a given example,
 see the `strict_min_version` part of the [applications key](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/applications)
 in the example's manifest.json file.
 
+## Installation
+
+There are a couple ways to try out the example extensions in this repository.
+
+1. Open Firefox and load `about:debugging` in the URL bar. Click the
+   [Load Temporary Add-on](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Temporary_Installation_in_Firefox)
```
Comment (wbamberg) [inline README.md]: Could we embed the video here? https://www.youtube.com/watch?v=cer9EUKegG4

File: README.md [hunk 2]
```diff
@@ -36,6 +36,22 @@ To check the minimum version of Firefox needed for a given example,
 see the `strict_min_version` part of the [applications key](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/applications)
 in the example's manifest.json file.
 
+## Installation
```
Comment (wbamberg) [inline README.md]: Could we call this "Installing an example"?

File: README.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ The examples are made available under the
 To use the repository, first clone it.
 
 Each example is in its own top-level directory. Install an example in your
-favourite web browser (see [installation instructions for Firefox](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Packaging_and_installation)),
+favourite web browser (installation instructions are below),
```
Comment (wbamberg) [inline README.md]: Can we link to this section?

File: README.md (hunk 2 above)
Comment (kumar303) [inline README.md]: Sure. Fixed.

File: README.md (hunk 3 above)
Comment (kumar303) [inline README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (wbamberg) [inline README.md]: Actually I wondered if we could embed it in the README itself, but a link is fine.

**Comment by wbamberg** [comment]
Thanks @kumar303 !

**Pull request merged**